spec/bolt_server/app_integration_spec.rb

```diff
@@ -99,5 +99,40 @@ def app
         expect(result['result']["stderr"]).to match(/not-a-command/)
       end
     end
+
+    describe "upload_file" do
+      let(:path) { '/ssh/upload_file' }
```

Comment:
do we need to worry about a discrete test for WinRM?

Reply:
I'm not sure if we're set up for that, as the other endpoints aren't tested that way.

Reply:
So does anywhere you've looked test WinRM vs SSH in this type of context?

Reply:
Maybe that's better handled on the orchestrator acceptance/integration side? @McdonaldSeanp ideas here?

Reply:
The docker tests in this file won't be compatible with Windows, so testing likely needs a more full stack.

Reply:
No, we don't need discrete unit testing for WinRM. There are no differences in this level of the code between ssh/winrm as far as the execution goes.

Acceptance testing for bolt will cover that the bolt level code (the level below this puma API) that we call here works over WinRM, and acceptance testing for orchestrator will test the integration of everything works for WinRM hosts.